spec/features/create_dataset_spec.rb

```diff
@@ -7,11 +7,11 @@
 
   context "when the user goes through entire flow" do
     before(:each) do
-      sign_in_user
+      sign_in_as(user)
     end
 
     it "navigates to new dataset form" do
-      expect(page).to have_current_path("/tasks")
```

Comment:
The tasks behaviour should be unaffected by using signon.

Reply:
Unfortunately the home page and related pages are no longer relevant, which is consistent with other GOV.UK admin apps. Jon's happy for now that we redirect to the tasks page, which is consistent with the current behaviour when signing-in.